torch/_python_dispatcher.py

```diff
@@ -48,14 +49,20 @@
 This file only provides the simplified API for developers, revelant test code is located in
 test/test_dispatch.py
 """
+
+
 class PythonDispatcher:
     namespace = "__test__"
     name = "foo"
     runtime_keys = [
-        "CPU", "AutogradCPU",
-        "FPGA", "AutogradOther",
-        "XLA", "AutogradXLA",
-        "Lazy", "AutogradLazy",
+        "CPU",
+        "AutogradCPU",
+        "FPGA",
+        "AutogradOther",
+        "XLA",
+        "AutogradXLA",
+        "Lazy",
+        "AutogradLazy",
```

Comment:
Someone carefully aligned the keys to convey relationship, and after autoformat it's gone.

Reply:
You're right in this case. The original format is easier to read. I think we can make an exception for this one (and any others that is standing out like this)

Reply:
For this specific case, I will add `# fmt: off` and `# fmt: on` to keep it as before. That's also the general way to tell BLACK to not format certain part of the code base.